index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main">
```

Comment:
- I think it would be better to use the `<main>` tag that is already part of HTML instead of the ID given that this is the main content of this page. It's a good practice to use tags for purposes of code readability and styling. Read more about that [here](https://www.thoughtco.com/why-use-semantic-html-3468271#:~:text=By%20adding%20semantic%20tags%20to,page%20and%20its%20content%20is.)

Reply:
- [ ] Please use [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) elements as they clearly describe their meaning to both the browser and the developer.